src/Features/CSharpTest/SemanticSearch/CSharpSemanticSearchServiceTests.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.SemanticSearch;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.SemanticSearch;
+
+[UseExportProvider]
+public sealed class CSharpSemanticSearchServiceTests
```

Comment:
since semantic-search is its own whole set of projects, should it have dedicated test projects as well?

Reply:
It is not a separate set of projects. It's implemented as any other IDE feature in Features and VS layers. The extra projects are there only to produce reference assemblies.